manage_breast_screening/clinics/tests/test_models.py

```diff
@@ -38,37 +39,77 @@ def test_status_filtering():
     assertQuerySetEqual(models.Clinic.objects.completed(), {past}, ordered=False)
 
 
-@pytest.mark.django_db
-def test_user_assignment_creation():
-    assignment = UserAssignmentFactory()
-    assert assignment.user is not None
-    assert assignment.provider is not None
-    assert assignment.pk is not None
-
-
-@pytest.mark.django_db
-def test_user_assignment_str():
-    user = UserFactory(first_name="John", last_name="Doe")
-    provider = ProviderFactory(name="Test Provider")
-    assignment = UserAssignmentFactory(user=user, provider=provider)
-    assert str(assignment) == "John Doe → Test Provider"
-
+class TestUserAssignment:
+    @pytest.mark.django_db
```

Comment:
I think the database is not needed for this case, and you could use the build strategy